blocks/library/embed/index.js

```diff
@@ -207,7 +207,7 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 						/>
 					</div>
 				);
-				let typeClassName = 'wp-block-embed';
+				let typeClassName = classnames( className, 'wp-block-embed' );
```

Comment:
Should we rename this variable to classNames? As now it is not type specific and can also contain our custom class.
Given that we are now using classnames we can refactor the if condition bellow to be included in the classnames logic. wp-block-embed can also be removed because the className prop of the embed block already includes it.
So I think this can be simplified into:
const classNames = classnames( className, { 'is-video': 'video' === type } );

Reply:
@jorgefilipecosta I like that refactor. Then, does it make sense to have the classNames constant at all? For instance, the paragraph block inlines that logic directly in the return.

Reply:
Yes, I think inlining like the paragraph is a good idea :)